plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java

```diff
@@ -165,7 +165,8 @@ public Date getLastTimeLoaded() {
     @RequirePOST
     @Restricted(NoExternalUse.class)
     public void doReload(StaplerRequest request, StaplerResponse response) throws Exception {
-        if (!Jenkins.get().hasPermission(Jenkins.ADMINISTER)) {
+        final Jenkins jenkins = Jenkins.get();
+        if (!jenkins.hasPermission(Jenkins.ADMINISTER) && !jenkins.hasPermission(Jenkins.MANAGE)) {
```

Comment:
Not sure whether I should test for both. 
Allowing for users with Manage rights should work for users with Administer rights too but since I can't test it (cf PR description) I can't confirm this and this is not explained explicitly in the documentation.

Reply:
it's implied
```suggestion
        if (!Jenkins.get().hasPermission(Jenkins.MANAGE)) {
```